src/o365/outlook/commands.ts

```diff
@@ -2,5 +2,6 @@ const prefix: string = 'outlook';
 
 export default {
   OUTLOOK_SENDMAIL: `${prefix} sendmail`,
-  OUTLOOK_MAIL_SEND: `${prefix} mail send`
+  OUTLOOK_MAIL_SEND: `${prefix} mail send`,
+  OUTLOOK_USAGE_GETMAILBOXUSAGESTORAGE: `${prefix} usage getmailboxusagestorage`
```

Comment:
Could you please replace `usage` with `report` in both? So, it should change to 
```
OUTLOOK_REPORT_GETMAILBOXUSAGESTORAGE: `${prefix} report getmailboxusagestorage`
```

Reply:
@plamber, @waldekmastykarz should we take away the `get` to reduce the length of the command so instead of
 `outlook report getmailboxusagestorage` to be  `outlook report mailboxusagestorage` ?

We can, of course, keep it consistent with the endpoint, but from my experience, this is just more cognitive load for the user.

Reply:
hi,
this makes definitely sense. I am removing the "get" and I will replace the usage term to remain consistent.

br,